docs/docsite/rst/contributing/index.rst

```diff
@@ -8,7 +8,7 @@ Contributing Content
    :maxdepth: 2
 
    creating_role
-   creating_multi
+   creating_collections
    creating_apb
    version
```

Comment:
The `version` doc needs to be updated to describe versioning Roles and APBs vs Collections